---
title: "On the use of relative brain size"
author: "Simeon Q. Smeele $^{1,2,3}$"
output:
  pdf_document: default
  word_document: default
header-includes:
- \usepackage{lineno}
- \linenumbers
- \usepackage{setspace}
- \doublespacing
editor_options:
  chunk_output_type: console
bibliography: bibliography.bib
---

```{r include=FALSE}
# Libraries
## for installation instructions see the README
library(cmdstanr)
library(scales)

# Paths
path_response_stan = 'models/response.stan' # models
path_predictor_stan = 'models/predictor.stan'
path_sem_stan = 'models/sem.stan'
path_pdf_response = 'figures/response.pdf' # pdfs
path_pdf_predictor = 'figures/predictor.pdf'
path_pdf_DAG = 'figures/DAG.pdf'
```

$^1$Cognitive & Cultural Ecology Research Group, Max Planck Institute of Animal Behavior, Radolfzell, Germany

$^2$Department of Human Behavior, Ecology and Culture, Max Planck Institute for Evolutionary
Anthropology, Leipzig, Germany

$^3$Department of Biology, University of Konstanz, Konstanz, Germany

**Keywords:** Bayesian statistics, comparative analysis, multiple regression, relative brain size, structural equation model

**Author for correspondence:** Simeon Q. Smeele, ssmeele@ab.mpg.de

# Abstract

1. There is a long standing interest in the effect of relative brain size on other life history variables in a comparative context. Historically, residuals have been used to calculate these effects, but more recently it has been recognised that regression on residuals is not good practice. Instead, absolute brain size and body size are included in a multiple regression, with the idea that this controls for allometry.

2. I use a simple simulation to illustrate how a case with brain size as response variable differs from a case with relative brain size as predictor variable. I use the simulated data to test which modelling approach can estimate the underlying causal effects for each case.

3. The results show that a multiple regression models with both body size and another variable as predictor variable and brain size as response variable work well. However, if relative brain size is a predictor variable, a multiple regression fails to correctly estimate the effects. 

4. I propose the use of structural equation models to simultaneously estimate relative brain size and its effect on the third variable and discuss other potential methods.

# Introduction

Brain size is often theorised to be important in the life history of species and has been linked to everything from innovation rates [@lefebvre2004brains], to sociality [@dunbar2007evolution] and longevity [@minias2017longevity]. It is important to note that it is often the *relative* and not the *absolute* size of the brain that is of interest. To control for allometric scaling (larger species have larger brains), brain size is measured relative to body size. There exists a large diversity of relative brain size measures [@healy2007critique], but some decades ago the use of residual brain size was proposed and quickly became the most popular [@jerison1973evolution]. This measure is simply the residual from a model that has log body size as predictor and log brain size as response variable. It is an intuitively attractive approach, since it seems to get at *that extra bit of brain* a species has. Multiple challenges were recognised early on. Probably the most debated is the effect of phylogeny [@armstrong1983relative]. When comparing across multiple species nested at different levels of taxonomy, how does one estimate the 'true' slope of the model? Several techniques have been proposed to mitigate this, but the debate is ongoing [@font2019rethinking; @burger2019allometry]. 

More recently another statistical caveat has been highlighted (for an overview see @freckleton2002misuse). The use of residuals is great for visualisation, but should not be used as data. The proposed solution is to include body size in a multiple regression, which is very similar to the use of residual brain size, if - and only if - relative brain size is the response variable. This approach takes care of both the phylogenetic signal (since the model can include the phylogenetic variance-covariance matrix) and correct estimation of uncertainty. However, one major issue has been overlooked: if relative brain size is a predictor variable, including body size as a second predictor leads to incorrect inference. This becomes clear when looking at the causal structure of such a case. The easiest way to decide which variables to include in an analysis is to draw a Directed Acyclic Graph (DAG). Here all variables are connected with arrows which indicate the direction of causality [@glymour2016causal]. 

To point out the difference between the case with relative brain size as response vs predictor variable, I will present the underlying assumed causal structure presented in Figure 1. To avoid discussion about directionality I have simply named the third variable *z*. In the first case both body size and *z* cause brain size (see Figure 1a). To decide how to best estimate the direct effect of *z* on brain size, one needs to make sure that all *back-door paths* are closed. In other words, all arrows that points towards *z* need to be considered. But since there are no such arrows in this case, there is no need to include additional variables. However, body size explains most of the variation in brain size. Therefore it is still a good idea to include this variable to get a more precise estimate of the effect of *z* on brain size. Or in other words on relative brain size, since allometry is now accounted for.

The second case is causally more opaque (see Figure 1b). In general the question of interest is how cognitive ability influences the third variable *z*. But since this cannot be measured reliably across species (therefore denoted by a U for unobserved), relative brain size is used as a proxy. This is where it becomes tricky. The assumption is that it is the *extra bit of brain* that is caused by the need for cognitive ability. And it is therefore *relative* brain size that needs to be included as proxy. The variable we initially measured is *absolute* brain size, and is a collider in this DAG. A collider is a variable that is caused by two or more other variables. When including a collider in the analysis it *opens up* a back-door path, in our case through body size. This is problematic, because the estimated effect of brain size on *z* now also contains some of the effect of body size on *z*. Therefore the estimated effect of body size on *z* will be biased. There is no way to correct for this using the variables in the DAG in a multiple regression. To include both relative brain size and body size one needs a system that contains regressions with brain size as response (of body size) and as predictor (of the third variable).

A structural equation model is such a system. It contains regressions for each variable and allows brain size to be response and predictor variable simultaneously [@bowen2011structural]. When fitted using a Bayesian approach, information flows in both directions, since the likelihood is computed for the whole system at each step. The aim of this paper is to show the estimation error in multiple linear regressions using a simulation and propose a simple Bayesian structural equation model as a solution that can be easily adapted to most comparative studies. I also provide a version of the structural equation model with *ulam* from the *rethinking* package as front-end [@rethinking] so that models can be adapted within the R environment. 

```{r dag, echo=FALSE, out.width = "100%", fig.cap= "Directed Acyclic Graph of the two cases. Arrows indicatethe direction of causality. z stands for the third variable of interest. a) Case I: brain size is the reponse variable. b) Case II: relative brain size is a proxy predictor variable. U stands for the unobserved variable cognitive ability."}
knitr::include_graphics(path_pdf_DAG)
```

# Methods

To show the difference between a case where relative brain size is the *response* variable and where it is a *predictor* variable I simulated two simple cases with three variables. To simplify the simulation I have not attempted to simulate realistic values for body size and relative brain size, but just used values with mean = 0 and standard deviation = 1. This allows me to draw general conclusions that are not sensitive to the scale of the variables.

I simulated 20 datasets per case and present parameter estimates for all datasets. Frequentist linear models were fitted with the *lm* function from R [@R]. Bayesian models were fitted using the *cmstanr* package [@cmdstanr], which runs the No U-turn Sampler in Stan [@gelman2015stan] with four chains and default sampler settings (1000 warmup and 1000 sampling iterations). Rhat, divergent transitions and effective sample size were monitored by the package and any issues were reported. 

To test if sample size had an effect on which model performed best, additional simulations were run and analysed for 20 and 1000 species (see supplemental results). To test if more or less informative priors had an affect on which model performed best, and the performance of the best model, additional simulations were run with informative and vague priors (see supplemental materials).

## Case I: relative brain size as response variable

In the first case absolute brain size is caused by body size and *z*. The interest of the study is to what extent *z* causes additional increase in brain size. In other words, to what extend *z* correlates with relative brain size. I simulated 20 data sets with 100 species with the following structure:

$$\begin{aligned}
\text{body size} & \sim \text{normal}(0,\ 1) \\
z & \sim \text{normal}(0,\ 1) \\
\text{brain size} & \sim \text{normal}(\mu_{brain},\ 1) \\
\mu_{brain} & = 1 * \text{body size} + 1 * z
\end{aligned}$$

I analysed the resulting data with a frequentist linear model and with the Bayesian equivalent. Then I plotted the estimated effect of body size and *z* to show how well parameters were retrieved. 

## Case II: relative brain size as predictor variable

In the second case both body size and relative brain size are a predictor of *z*. For simplicity I removed the unobserved variable from the simulation, and instead included a direct effect of relative brain size on z and absolute brain size. This would still be realistic when considering developmental time as a function of body size and relative brain size. The interest of the study is therefore to what extend relative brain size causes *z*. I simulated 20 data sets with 100 species with the following structure:

$$\begin{aligned}
\text{body size} & \sim \text{normal}(0,\ 1) \\
\text{relative brain size} & \sim \text{normal}(0,\ 1) \\
\text{brain size} & = 1 * \text{relative brain size} + 1 * \text{body size} \\
z & \sim \text{normal}(\mu_z,\ 1) \\
\mu_z & = 1 * \text{body size} + 1 * \text{relative brain size}
\end{aligned}$$

I analysed the resulting data with both a frequentist and Bayesian linear model where brain size and body size were included as predictor variables. Additionally I analysed the data with a Bayesian structural equation model that included sub-models for all causal paths:

$$\begin{aligned}
\text{body size} & \sim \text{normal}(\alpha_\text{body},\ \sigma_\text{body}) \\
\\
\text{brain size} & \sim \text{normal}(\mu_\text{brain},\ \sigma_\text{brain}) \\
\mu_\text{brain[i]} & = \alpha_\text{brain} + \beta_\text{body} * \text{body size}_\text{[i]} \\
\\
z & \sim \text{normal}(\mu_z,\ \sigma_z) \\
\mu_{z\text{[i]}} & = \alpha_z + \gamma_\text{body} * \text{body size}_\text{[i]} + \gamma_\text{brain} * (\text{brain size}_\text{[i]} - \mu_\text{brain[i]})\\
\\
\alpha_\text{body},\ \alpha_\text{brain},\ \alpha_z  & \sim \text{normal}(0,\ 1)\\
\beta_\text{body},\ \gamma_\text{body},\ \gamma_\text{brain} & \sim \text{normal}(0,\ 1)\\
\sigma_\text{body},\ \sigma_\text{brain},\ \sigma_z & \sim \text{exponential}(1)\\
\end{aligned}$$

The model includes a regression for each variable. Body size is not a function of any variable. Brain size is a function of body size. *z* is a function of body size and relative brain size (where relative brain size is the difference between the actual and predicted brain size). Relative brain size in this model is very similar to residual brain size, but since it is computed at each iteration information flows in both directions and measurement error is correctly estimated. The last three lines of the model are the priors for all parameters. Note that the priors for the slopes ($\beta$ and $\gamma$) are set to normal(0, 1), which regularises them slightly and assumes no effect of the predictors. For empirical studies theory might provide more informative priors, which would further increase the accuracy of the model. 

# Results

For Case I, where relative brain size was the response variable, both models estimated the parameters very well (see Figure 2). For Case II, where relative brain size was a predictor variable, the effect of brain size was estimated well by all models, but the effect of body size was only estimated correctly by the structural equation model (see Figure 3). Both the frequentist and Bayesian linear model estimated the effect of body mass to be essentially 0.

Results were similar for simulations with smaller and larger sample sizes (see supplemental materials, Figure S1-4). Results were also similar for models with vague and informative prior (see supplemental materials, Figure S5-8). The informative priors constrained the Bayesian linear models estimate of the body size effect to positive values, but none of the posterior distributions included the true value of the body size effect. In other words, even if the model is run with the correct value as prior, it estimates the effect to be absent.  

```{r response, include=FALSE}
# Settings
set.seed(1)
N = 100
z_effect = 1
body_effect = 1
model_response = cmdstan_model(path_response_stan)

# Simulate data 20 times
out = lapply(1:20, function(i){
  body_size = rnorm(N, 1, 1)
  z = rnorm(N, 0, 1)
  mu_brain = body_effect * body_size + z_effect * z
  brain_size = rnorm(N, mu_brain, 1)
  
  # Run linear model
  linear_fit = lm(brain_size ~ body_size + z)
  
  # Run Bayesian linear model
  d = list(N = N, brain_size = brain_size, body_size = body_size, z = z) # save data as list
  bay_linear_fit = model_response$sample(data = d, # run model
                                         seed = 1, 
                                         chains = 4, 
                                         parallel_chains = 4, # runs on four threads
                                         refresh = 2000)
  post_linear_fit = bay_linear_fit$draws(format = 'draws_df') # draw posterior samples
  
  return(list(linear_fit = linear_fit, post_linear_fit = post_linear_fit))
})

# Plot the fit
pdf(path_pdf_response, 7, 5.5)
par(mfrow = c(2,2), oma = c(1, 4, 1, 1), mar = c(4, 1, 2, 3))
plot(NULL, xlim = c(-0.5, 1.5), ylim = c(0, 5), xlab = '', yaxt = 'n', main = 'linear model')
axis(4)
mtext('density', 4, 2.5)
mtext('effect z', 1, 2.5)
abline(v = z_effect, lty = 2, lwd = 5, col = alpha('black', 0.5))
for(i in 1:20) lines(density(rnorm(1e5, out[[i]]$linear_fit$coefficients[3],
                                   summary(out[[i]]$linear_fit)$coefficients[3,2])), 
                     pch = 16, cex = 3, col = alpha('#D68910', 0.5))
mtext('z', 2, 3, font = 2, las = 2, adj = 0)
plot(NULL, xlim = c(-0.5, 1.5), ylim = c(0, 5), 
     xlab = '', yaxt = 'n', main = 'Bayesian linear model')
abline(v = z_effect, lty = 2, lwd = 5, col = alpha('black', 0.5))
for(i in 1:20) lines(density(out[[i]]$post_linear_fit$b_z), pch = 16, cex = 3, col = alpha('#7D3C98', 0.5))
axis(4)
mtext('density', 4, 2.5)
mtext('effect z', 1, 2.5)

plot(NULL, xlim = c(-0.5, 1.5), ylim = c(0, 5), 
     xlab = '', yaxt = 'n', main = '')
abline(v = body_effect, lty = 2, lwd = 5, col = alpha('black', 0.5))
for(i in 1:20) lines(density(rnorm(1e5, out[[i]]$linear_fit$coefficients[2], 
                                   summary(out[[i]]$linear_fit)$coefficients[2,2])), 
                     pch = 16, cex = 3, col = alpha('#D68910', 0.5))
mtext('body', 2, 3, font = 2, las = 2, adj = 0)
axis(4)
mtext('density', 4, 2.5)
mtext('effect body size', 1, 2.5)
plot(NULL, xlim = c(-0.5, 1.5), ylim = c(0, 5), 
     xlab = '', yaxt = 'n', main = '')
abline(v = body_effect, lty = 2, lwd = 5, col = alpha('black', 0.5))
for(i in 1:20) lines(density(out[[i]]$post_linear_fit$b_body), pch = 16, cex = 3, col = alpha('#7D3C98', 0.5))
axis(4)
mtext('density', 4, 2.5)
mtext('effect body size', 1, 2.5)
dev.off()
```

```{r plot response, echo=FALSE, out.width = "72%", fig.cap= "Parameter estimates from the linear model and Bayesian linear model with brain size as response variable. Dashed grey line is the true value. Orange density plots are normal distributions based on the mean and SE from the linear model. Purple density plots are the posterior distributions from the Bayesian model."}
knitr::include_graphics(path_pdf_response)
```

```{r predictor, include=FALSE}
# Settings
set.seed(1)
N = 100
brain_effect = 1
body_effect = 1
model_predictor = cmdstan_model(path_predictor_stan)
model_sem = cmdstan_model(path_sem_stan)

# Simulate data
out = lapply(1:20, function(i){
  body_size = rnorm(N, 0, 1)
  rel_brain_size = rnorm(N, 0, 1)
  brain_size = 1 * body_size + rel_brain_size
  mu_z = body_effect * body_size + brain_effect * rel_brain_size
  z = rnorm(N, mu_z, 1)
  
  # Run linear model
  linear_fit = lm(z ~ body_size + brain_size)
  
  # Run Bayesian linear model
  d = list(N = N, brain_size = brain_size, body_size = body_size, z = z)
  bay_linear_fit = model_predictor$sample(data = d, 
                                          seed = 1, 
                                          chains = 4, 
                                          parallel_chains = 4,
                                          refresh = 2000)
  post_linear_fit = bay_linear_fit$draws(format = 'draws_df')
  
  # Run Bayesian SEM
  bay_sem_fit = model_sem$sample(data = d, 
                                 seed = 1, 
                                 chains = 4, 
                                 parallel_chains = 4,
                                 refresh = 2000)
  post_sem_fit = bay_sem_fit$draws(format = 'draws_df')
  
  return(list(linear_fit = linear_fit, 
              post_linear_fit = post_linear_fit, 
              post_sem_fit = post_sem_fit))
})

# Plot the fit
pdf(path_pdf_predictor, 9, 4.8)
par(mfrow = c(2,3), oma = c(1, 4, 1, 1), mar = c(4, 1, 2, 3))
plot(NULL, xlim = c(-0.5, 1.5), ylim = c(0, 5), 
     xlab = '', yaxt = 'n', main = 'linear model')
abline(v = brain_effect, lty = 2, lwd = 5, col = alpha('black', 0.5))
for(i in 1:20) lines(density(rnorm(1e5, out[[i]]$linear_fit$coefficients[3],
                                   summary(out[[i]]$linear_fit)$coefficients[3,2])), 
                     pch = 16, cex = 3, col = alpha('#D68910', 0.5))
axis(4)
mtext('density', 4, 2.5)
mtext('effect brain size', 1, 2.5)
mtext('brain', 2, 4, font = 2, las = 2, adj = 0)
plot(NULL, xlim = c(-0.5, 1.5), ylim = c(0, 5), 
     xlab = '', yaxt = 'n', main = 'Bayesian linear model')
abline(v = brain_effect, lty = 2, lwd = 5, col = alpha('black', 0.5))
for(i in 1:20) lines(density(out[[i]]$post_linear_fit$b_brain), pch = 16, cex = 3, col = alpha('#7D3C98', 0.5))
axis(4)
mtext('density', 4, 2.5)
mtext('effect brain size', 1, 2.5)
plot(NULL, xlim = c(-0.5, 1.5), ylim = c(0, 5), 
     xlab = '', yaxt = 'n', main = 'Bayesian SEM model')
abline(v = brain_effect, lty = 2, lwd = 5, col = alpha('black', 0.5))
for(i in 1:20) lines(density(out[[i]]$post_sem_fit$b_brain_z), pch = 16, cex = 3, col = alpha('#16A085', 0.5))
axis(4)
mtext('density', 4, 2.5)
mtext('effect brain size', 1, 2.5)

plot(NULL, xlim = c(-0.5, 1.5), ylim = c(0, 5), 
     xlab = '', yaxt = 'n', main = '')
abline(v = body_effect, lty = 2, lwd = 5, col = alpha('black', 0.5))
for(i in 1:20) lines(density(rnorm(1e5, out[[i]]$linear_fit$coefficients[2],
                                   summary(out[[i]]$linear_fit)$coefficients[2,2])), 
                     pch = 16, cex = 3, col = alpha('#D68910', 0.5))
axis(4)
mtext('density', 4, 2.5)
mtext('effect body size', 1, 2.5)
mtext('body', 2, 4, font = 2, las = 2, adj = 0)
plot(NULL, xlim = c(-0.5, 1.5), ylim = c(0, 5), 
     xlab = '', yaxt = 'n', main = '')
abline(v = body_effect, lty = 2, lwd = 5, col = alpha('black', 0.5))
for(i in 1:20) lines(density(out[[i]]$post_linear_fit$b_body), pch = 16, cex = 3, col = alpha('#7D3C98', 0.5))
axis(4)
mtext('density', 4, 2.5)
mtext('effect body size', 1, 2.5)
plot(NULL, xlim = c(-0.5, 1.5), ylim = c(0, 5), 
     xlab = '', yaxt = 'n', main = '')
abline(v = body_effect, lty = 2, lwd = 5, col = alpha('black', 0.5))
for(i in 1:20) lines(density(out[[i]]$post_sem_fit$b_body_z), pch = 16, cex = 3, col = alpha('#16A085', 0.5))
axis(4)
mtext('density', 4, 2.5)
mtext('effect body size', 1, 2.5)
dev.off()
```

```{r plot predictor, echo=FALSE, out.width = "100%", fig.cap= "Parameter estimates from the linear model, Bayesian linear model and Bayesian strucutral equation model with relative brain size as predictor variable. Dashed grey line is the true value. Orange density plots are normal distributions based on the mean and SE from the linear model. Purple and green density plots are the posterior distributions from the Bayesian models."}
knitr::include_graphics(path_pdf_predictor)
```

# Discussion 

Several recent studies have claimed to study the effect of relative brain size by including both absolute brain size and body size as predictors [@street2017coevolution; @gonzalez2010large; @isler2009why]. These studies also reported the direct effect of body size and sometimes drew conclusions based on the sign of this effect. Since all these studies used some version of a linear model (be it phylogenetic and/or Bayesian), they actually tested the effect of absolute brain size, since including body size only accounts for allometry in the response variable. The simulations in this paper showed that including body size as additional predictor to control for allometric scaling of brain size works well if relative brain size is the response variable, but not if relative brain size is a predictor variable. Perhaps counter-intuitively, the effect of brain size was still estimated correctly by all models. It was the body size effect that was biased in the linear models. The use of such models would lead to the incorrect inference that there was no effect of body size at all. 

One way to create some intuition about what is going on is that absolute brain size (which was the actual predictor variable included in the linear models) contains information about both body size and relative brain size. In a sense this variable controls for the body size effect already, leaving the direct effect of body size zero. In a case where body size itself does not have an effect on *z*, it does not actually need to be included at all. Using absolute brain size would be fine. The variation in brain size due to allometric scaling would just create noise. Using absolute brain size would of course lead to a less precise estimate of the effect of brain size, so using relative brain size from a structural equation model would still be preferable.  

Structural equation models are similar to phylogenetic path analysis [@gonzalez2014introduction] and multivariate regressions [@izenman2013multivariate]. However, both these techniques do not allow the inclusion of the difference between the observed and predicted brain size from one model as predictor for a second model. A potentially even more powerful approach was put forward by @smaers2009inferring and involves reconstructing the ancestral states of both body size and brain size. Recently this approach was used to test brain evolution in mammals and it was shown that relatively large brains can be achieved by highly divergent paths [@smaers2021evolution]. The authors contest the notion that there is a universal scaling between body and brain size, and instead propose that relatively large brains can just as well be a result of selection towards smaller bodies. Despite this observation, there might still be value in the use of relative brain size for two reasons. First, the largest non-cognitive effects were due to a shift in the slope of the body to brain relation. Such a shift would be less likely to affect results when considering a lower order taxonomic group (e.g., only primates) and could be partially accounted for by including multiple slopes (e.g., one per family). Second, running a Bayesian multiregime OU modeling approach is not straightforward and becomes really difficult when many covariates are included. The best compromise is to first study the allometric patterns in the taxon of interest and then decide if residual brain size can be used as proxy for cognitive ability. 

For this paper I drew all variables from normal distributions with zero mean and standard deviation one. I furthermore simulated *z* as a function of relative brain size, rather than simulating both *z* and brain size as a function of the unobserved cognitive ability (as depicted in Figure 1b). I chose to do this to illustrate that even under these simple conditions a multiple linear regression cannot estimate the effect of body size correctly. In empirical studies the exact causal structure will be different, but the general observation that one cannot control for allometry in a multiple linear regression when brain size is not the response variable still stands. A regular confounder check as described by @laubach2021biologist should inform the design of the model. One example of relative brain size as predictor variable is the study of life history and innovativeness. @sol2016life suggested that relative brain size predicts both the innovation propensity and maximum life span of birds. They recognised the critique on using residuals, but still used this approach, because they wanted to remove the allometric effects from brain size and not from innovation propensity. This was a valid argument since the original critique assumed a case where $x_1$ and $x_2$ are both causing $y$ [@freckleton2002misuse]. When $x_1$ and $x_2$ are correlated, the use of a residual of $y \sim x_1$ creates a bias. In the case of life history and innovativeness the causal structure is different and this particular problem does not arise. However, in an empirical study measurement error, missing data and phylogenetic covariance still need to be accounted for. When using residuals, uncertainty does not propagate because models are run sequentially. 

Structural equation models allow for the inclusion of measurement error, imputation of missing data and phylogenetic coveriance [@smeele2022coevolution]. Furthermore, the Bayesian implementation is useful in cases with low sample size. Prior information about parameters can be used to increase accuracy. E.g., on average brain size can only increase with body size. The prior for this relationship should therefore be set to only include positive values. This creates a clear advantage over frequentist linear models, which allow any real number and can therefore produce scientifically nonsensical results. 

# Conclusion

The aim of this paper was to show how a case where relative brain size is a response variable differs from a case where it is a predictor variable. From a simple simulation it becomes clear that one cannot include absolute brain size and body size as covariates if relative brain size is theorised to predict the third variable. Multiple other approaches can be used instead and the design of the model should be based on the assumed causal structure of the system. I provide a simple structural equation model, which works for the simulated data. Future studies should make sure all back-door paths are closed for their DAG and potentially include additional components to control for measurement error, missing data and phylogeny.  The use of these models is not limited to relative brain size, but can be used for any comparative study in which multiple causal paths are of interest.   

**Acknowledgements.** I would like to thank all anonymous reviewers for valuable input. Their contribution helped to clarify the manuscript and gain new insights. 

**Data accessibility.** All data is generated in the simulation. Code is publicly available at: https://github.com/simeonqs/On_the_use_of_relative_brain_size.

**Competing interests.** I declare I have no competing interests.

**Funding.** I received funding from the International Max Planck Research School for Quantitative Behaviour, Ecology and Evolution. 

**ORCID:** 0000-0003-1001-6615

# References
